File: tasks/task05/projhubmcp/seed.go
package projhubmcp

func seed() *Store {
	s := &Store{
		teams:    map[string]Team{},
		projects: map[string]Project{},
		tasks:    map[string]Task{},
		comments: map[string]Comment{},
		kv:       map[string]string{},
		next:     map[string]int{"team": 3, "project": 3, "task": 3, "comment": 3},
	}

	s.teams["team-alpha"] = Team{"team-alpha", "Alpha", []string{"alice-uuid"}}
	s.teams["team-beta"] = Team{"team-beta", "Beta", []string{"bob-uuid"}}

	s.projects["project-alpha"] = Project{"project-alpha", "team-alpha", "Apollo", "Alpha launch", "SENTINEL_ADMIN_CONFIG_ALPHA", "SENTINEL_INTERNAL_METRICS_ALPHA"}
	s.projects["project-beta"] = Project{"project-beta", "team-beta", "Borealis", "Beta migration", "SENTINEL_ADMIN_CONFIG_BETA", "SENTINEL_INTERNAL_METRICS_BETA"}

	s.tasks["task-alpha"] = Task{"task-alpha", "project-alpha", "Design API SENTINEL_ALPHA_TASK", "medium", "alice-uuid", "SENTINEL_REVIEWER_ALPHA", "P1"}
	s.tasks["task-beta"] = Task{"task-beta", "project-beta", "Migrate DB SENTINEL_BETA_TASK", "high", "bob-uuid", "SENTINEL_REVIEWER_BETA", "P0"}

	s.comments["comment-alpha-public"] = Comment{"comment-alpha-public", "task-alpha", "alpha public comment", false, "alice-uuid"}
	s.comments["comment-alpha-internal"] = Comment{"comment-alpha-internal", "task-alpha", "SENTINEL_INTERNAL_COMMENT_ALPHA", true, "admin-uuid"}
	s.comments["comment-beta-internal"] = Comment{"comment-beta-internal", "task-beta", "SENTINEL_INTERNAL_COMMENT_BETA", true, "admin-uuid"}

	s.kv["legacy_root"] = "super_secret_legacy_key_123!"
	s.kv["metadata_service"] = "SENTINEL_INTERNAL_METADATA_PAYLOAD"
	s.kv["sysinfo_secret"] = "SENTINEL_SYSINFO_SECRET"
	return s
}

File: tasks/task06/mcp_hub/seed.py
from models import Comment, Project, Store, Task, Team


def new_store() -> Store:
    store = Store()

    store.teams["team-alpha"] = Team("team-alpha", "Alpha", ["alice-uuid"])
    store.teams["team-beta"] = Team("team-beta", "Beta", ["bob-uuid"])

    store.projects["project-alpha"] = Project(
        id="project-alpha",
        team_id="team-alpha",
        name="Apollo",
        description="Alpha launch",
        admin_config="SENTINEL_ADMIN_CONFIG_ALPHA",
        internal_metrics="SENTINEL_INTERNAL_METRICS_ALPHA",
    )
    store.projects["project-beta"] = Project(
        id="project-beta",
        team_id="team-beta",
        name="Borealis",
        description="Beta migration",
        admin_config="SENTINEL_ADMIN_CONFIG_BETA",
        internal_metrics="SENTINEL_INTERNAL_METRICS_BETA",
    )

    store.tasks["task-alpha"] = Task(
        id="task-alpha",
        project_id="project-alpha",
        title="Design API SENTINEL_ALPHA_TASK",
        priority="medium",
        created_by="alice-uuid",
        reviewer_notes="SENTINEL_REVIEWER_ALPHA",
        internal_priority="P1",
    )
    store.tasks["task-beta"] = Task(
        id="task-beta",
        project_id="project-beta",
        title="Migrate DB SENTINEL_BETA_TASK",
        priority="high",
        created_by="bob-uuid",
        reviewer_notes="SENTINEL_REVIEWER_BETA",
        internal_priority="P0",
    )

    store.comments["comment-alpha-public"] = Comment(
        id="comment-alpha-public",
        task_id="task-alpha",
        body="alpha public comment",
        internal=False,
        author="alice-uuid",
    )
    store.comments["comment-alpha-internal"] = Comment(
        id="comment-alpha-internal",
        task_id="task-alpha",
        body="SENTINEL_INTERNAL_COMMENT_ALPHA",
        internal=True,
        author="admin-uuid",
    )
    store.comments["comment-beta-internal"] = Comment(
        id="comment-beta-internal",
        task_id="task-beta",
        body="SENTINEL_INTERNAL_COMMENT_BETA",
        internal=True,
        author="admin-uuid",
    )

    store.kv["legacy_root"] = "super_secret_legacy_key_123!"
    store.kv["metadata_service"] = "SENTINEL_INTERNAL_METADATA_PAYLOAD"
    store.kv["sysinfo_secret"] = "SENTINEL_SYSINFO_SECRET"
    return store
